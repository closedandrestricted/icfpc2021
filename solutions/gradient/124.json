{"vertices":[[64,243],[34,217],[34,223],[11,188],[70,214],[76,198],[45,189],[64,46],[53,188],[89,53],[98,177],[125,201],[73,69],[75,165],[48,156],[95,47],[77,75],[45,127],[76,153],[75,100],[129,173],[50,163],[122,45],[148,77],[104,74],[24,169],[129,95],[59,131],[81,121],[20,133],[111,148],[132,71],[97,100],[2,152],[163,59],[63,102],[40,111],[153,87],[5,106],[114,54],[18,132],[41,83],[109,75],[73,92],[48,83],[164,67],[138,64]]}