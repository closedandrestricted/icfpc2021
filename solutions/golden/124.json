{"vertices":[[61,252],[39,219],[38,224],[10,195],[74,207],[71,171],[45,189],[66,46],[50,187],[91,54],[100,179],[127,203],[72,70],[77,167],[50,156],[104,50],[83,78],[45,128],[78,155],[74,101],[142,179],[52,164],[129,44],[179,83],[109,80],[25,167],[160,65],[64,132],[78,122],[20,129],[113,168],[135,71],[97,104],[3,150],[179,106],[60,103],[49,109],[155,89],[11,99],[109,69],[14,127],[40,82],[88,78],[99,129],[83,107],[159,66],[134,72]]}